Specs/NumberWithUnit/Chinese/CurrencyModel.json

```diff
@@ -334,23 +334,6 @@
       }
     ]
   },
-  {
-    "Input": "50元6角三分",
```

Comment:
Why remove this case? If not currently supported, but it will be supported in the future, mark it as not supported in all platforms.